openlibrary/templates/check_ins/check_in_prompt.html

```diff
@@ -17,7 +17,7 @@
 
 <div id="check-in-display-$(work_olid)" class="last-read-date $cond(last_read_date, '', 'hidden')">
   <span class="date-display">
-    <span class="read-label">Read: </span>
+    <span class="read-label">$_("Read: ")</span>
```

Comment:
Leaving the space out of the string makes this a little less error prone for translators.

```suggestion
    <span class="read-label">$_("Read:") </span>
```